docs/future-work/oss-communication-improvements/share-interesting-content.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Share Interesting Content
```

Comment:
Wouldn't this one also be an evergreen? It seems very vaguely defined, both in scope and type. I would put this in the general https://thousandbrainsproject.readme.io/docs/contributing page under "promote monty" since it doesn't seem like an item that will require a large time commitment.

Reply:
moved.